docs/quickstart.rst

```diff
@@ -323,6 +323,54 @@ Much like data objects, collection objects have a method :meth:`~WrightTools.col
 
 Collections can be saved inside of wt5 files, so be aware that :meth:`~WrightTools.open` may return a collection or a data object based on the contents of your wt5 file.
 
+Manipulating Your Data -- A 1D Plot Example
+----------------------
+Now that you have learned how to manipulate data provided within WrightTools itself, it is time to investigate manipulation of personal, experimental data. Above, data was already put in WrightTools and was in the .wt5 file format, which permitted use of 'wt.open()'. Here, this will no longer be the case, and other avenues must be explored for performing analysis in WrightTools. Below we will investigate plotting an infrared spectrum from a typical laboratory instrument (Bruker Tensor 27).
+
+First thing, you need to obtain some experimental data. We provide an example infrared spectrum for you to manipulate `here <https://raw.githubusercontent.com/wright-group/WrightTools/master/WrightTools/datasets/Tensor27/CuPCtS_powder_ATR.dpt>`_. Once you access this page, right click and save this file to your computer in an easily findable directory.
+
+Now, open up your favorite Python IDE so you can continue. First, make sure that WrightTools is imported:
+
+.. code-block:: python
+
+   >>> import WrightTools as wt
+
+
+The next step is easier than you think. Identify the location of your file. For example, on Windows, if your username is user and the file is on your Desktop, then as far as Python is concerned, the file location is
+
+.. code-block:: python
+
+   'C:\\Users\\user\\Desktop\\IR_spec.dpt'
+
+Note that I renamed the file to "IR_spec.dpt" to make it easier to plug into my script. I find it easiest to do this in Notepad (open in Notepad, then save as "all files" and name the file IR_spec.dpt")
+
+We now need to import the file and (importantly) define it as a variable / data object. If the file is not imported as a data object, WrightTools will be confused.
+
+To do this, we can use an import data command. Fortunately, the data object for a Bruker Tensor 27 Infrared Spectrometer is already programmed into WrightTools, which eases the import process.
+
+For simplicity, I will define the data object as "d". Then, to import this file into WrightTools, one simply types:
+
+.. code-block:: python
+
+   d = wt.data.from_Tensor27('C:\\Users\\user\\Desktop\\IR_spec.dpt')
```

Comment:
```suggestion
   >>> d = wt.data.from_Tensor27('C:\\Users\\user\\Desktop\\IR_spec.dpt')
```

Just consistency with adding the python prompt for things typed into a python shell